DataProcessing/datax-host/src/main/scala/datax/executor/ExecutorHelper.scala

```diff
@@ -0,0 +1,85 @@
+// *********************************************************************
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License
+// *********************************************************************
+package datax.executor
+
+import datax.config.ConfigManager
+import datax.constants.{BlobProperties, JobArgument}
+import datax.fs.HadoopClient
+import datax.host.SparkJarLoader
+import datax.securedsetting.KeyVaultClient
+import org.apache.log4j.LogManager
+import org.apache.spark.broadcast
+import org.apache.spark.sql.SparkSession
+
+object ExecutorHelper {
+  private val logger = LogManager.getLogger(this.getClass)
+
+  /***
+    * Create broadcast variable for blob storage account key
+    * @param path blob storage path
+    * @param spark SparkSession
+    */
+  def createBlobStorageKeyBroadcastVariable(path: String, spark : SparkSession): broadcast.Broadcast[String] ={
```

Comment:
Is this generic enough or is specific for Executor? If it's generic, am wondering if this can be moved to HadoopClient where I found similar helper methods.

Reply:
This is specific to executor. Since this method doesnot perform any hadoop operation we moved this method out of hadoop client to a seperate class